depends/hosts/android.mk

```diff
@@ -1,4 +1,27 @@
+ifeq ($(HOST),armv7a-linux-android)
+android_AR=$(ANDROID_TOOLCHAIN_BIN)/arm-linux-androideabi-ar
+android_CXX=$(ANDROID_TOOLCHAIN_BIN)/$(HOST)eabi$(ANDROID_API_LEVEL)-clang++
+android_CC=$(ANDROID_TOOLCHAIN_BIN)/$(HOST)eabi$(ANDROID_API_LEVEL)-clang
+android_RANLIB=$(ANDROID_TOOLCHAIN_BIN)/arm-linux-androideabi-ranlib
+endif
+
+ifeq ($(HOST),i686-linux-android)
```

Comment:
I just realised that the following three `ifeq` statements have exactly the same body. Basically only `armv7a-linux-android` is a special case. So would be much more logical to change this to: 
```
ifeq ($(HOST),armv7a-linux-android)
....
else
.... 
endif
```

Reply:
Done